requirements.txt

```diff
@@ -0,0 +1,7 @@
+google-cloud-tasks
```

Comment:
Kind of worth the runtime update just for this, dependency management soooo much nicer on next gen appengine